### ISSUE TITLE
can you explain this loss function

### ISSUE THREAD

**Issue by BenxiaHu** [snehamitra/SCARlink#5]
Hello, 
SCARlink looks great to predict gene expression and identify cell-type-specific enhancers.

I just read your SCARlink paper. I am a little confused about the loss function.
Would you like to explain it?

<img width="611" alt="image" src="https://github.com/snehamitra/SCARlink/assets/12387462/d3ee1853-fa81-4223-a088-65eb1a21070a">

Best,

**Comment by snehamitra** [comment]
Hi, is your question about the hyperparameter $\alpha$? It is the weight associated with l2 regularization of the regression coefficient vector $\mathbf{w}$. We tune the value of $\alpha$ during cross validation to obtain the best fit of the model. We use l2 regularization to shrink the weights of tiles that that are not putative enhancers. Also, the first part of the loss function is the Poisson loss.

**Comment by BenxiaHu** [comment]
thanks.
Xi * W+e is the predicted gene expression.
Yi * log(Xi * W+e) looks a little confused.
Would you like to explain Yi * log(Xi * W+e)?

**Comment by snehamitra** [comment]
It is derived from the likelihood function of the Poisson distribution. https://web.stanford.edu/class/archive/stats/stats200/stats200.1172/Lecture27.pdf

Closing this thread since it is not related to the implementation of SCARlink.

**Comment by BenxiaHu** [comment]
hello,
<img width="530" alt="image" src="https://github.com/snehamitra/SCARlink/assets/12387462/d7d6b9f9-9b62-4ae0-94a4-a3c519c77055">
I am still confused this model:
x and y are normliazed values, not count. why are they applied to poisson distribution?
